library/Backends/llvm/src/Juvix/Backends/LLVM/Compilation.hs

```diff
@@ -220,7 +235,7 @@ typeToLLVM ty@(ErasedAnn.Pi _usage _f _xs) =
   Types.pointerOf
     LLVM.FunctionType
       { LLVM.resultType = typeToLLVM resultType,
-        LLVM.argumentTypes = map typeToLLVM argumentTypes,
+        LLVM.argumentTypes = Juvix.Library.map typeToLLVM argumentTypes,
```

Comment:
Can't we just use `map` or `<$>`

Reply:
The new import of `HashMap` brought in another `map`, which is why I had to make the pre-existing one specify which `map` it was referring to.

Reply:
I think it's better to import HashMap qualified

Reply:
We should import it qualified, we should also change the `map` call to `fmap`

Reply:
I've now realized that the import was only required by an older version of the code, so I should simply remove it!